tools/test_utils.py

```diff
@@ -575,6 +575,162 @@ def run_resnet50_forward_pass_from_artifacts(ngraph_tf_src_dir, artifact_dir,
     os.chdir(root_pwd)
 
 
+# See https://github.com/tensorflow/benchmarks/tree/master/perfzero
+def run_tfperfzero_resnet50_infer_from_artifacts(
```

Comment:
What is this for?

Reply:
For future extension to use TF recommended benchmarking

Reply:
Maybe add this when we actually get around to testing/benchmarking with PerfZero?